src/components/EntityPage/File/index.js

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { injectState } from 'freactal';
+import { withTheme } from 'emotion-theming';
+
+import styled from 'react-emotion';
+import Row from '../../../uikit/Row';
+import Column from '../../../uikit/Column';
+
+import { withApi } from 'services/api';
+import {
+  EntityTitleBar,
+  EntityTitle,
+  EntityActionBar,
+  EntityContent,
+  EntityContentSection,
+  EntityContentDivider,
+} from '../';
+
+const enhance = compose(
+  injectState,
+  withTheme,
+  withApi,
+);
+
+const Container = styled(Column)`
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  align-items: center;
+`;
+
+const FileEntity = ({ state, effects, theme, fileId, ...props }) => {
+  return (
+    <Container>
+      <EntityTitleBar>
+        <EntityTitle icon="file" title={fileId} tags={['WGS', 'Aligned Reads']} />
+      </EntityTitleBar>
+      <EntityActionBar>Share Button</EntityActionBar>
+      <EntityContent>
+        <EntityContentSection title="File Properties">
+          <Row style={{ width: '100%' }}>
```

Comment:
The content here is a good candidate for a grid system instead of forced 1 row, 2 column as I wrote it.

Would leave this for whoever is implementing the content I think...